packages/server/src/tests/utilities/TestConfiguration.ts

```diff
@@ -471,6 +474,15 @@ export default class TestConfiguration {
     }
   }
 
+  async defaultToProduction(cb: () => Promise<unknown>) {
```

Comment:
Why not use `withProdApp`?

Reply:
I did not realise this was available - good to know!